RSDCatalog/RSDCatalog/Info.plist

```diff
@@ -24,6 +24,8 @@
 	<string>This app uses the camera to take pictures as an example of using the Image Picker.</string>
 	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
 	<string>This app uses the GPS sensor to measure relative distance travelled.</string>
+	<key>NSLocationAlwaysUsageDescription</key>
```

Comment:
Yeah, I'm pretty sure one is for iOS 10 and earlier and the other is for iOS 11 and later, but can't remember which is which. But if we support both then I guess we need both.